typings/index.d.ts

```diff
@@ -69,7 +69,20 @@ export interface ViewPagerProps extends ReactNative.ViewProps {
      */
     pageMargin?: number;
 
-    onMoveShouldSetResponderCapture?: (event: any) => boolean;
+    style?: ReactNative.StyleProp<ReactNative.ViewStyle>
+
+    children?: React.ReactChildren
+
```

Comment:
I've got this, since this update
"Type 'Element[]' is not assignable to type 'ReactChildren'."

Reply:
can you try with ReactNode ? 

I found it here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1333

Reply:
No problem with ReactNode, sry for delay, didn't see your comment.